main.css

```diff
@@ -0,0 +1,569 @@
+  @apply text-[#A5A6AC];
+}
+.link--light {
+  @apply text-[#A5A6AC];
+}
+
+/* buttons */
+.btn, .header__btn, .cta__btn, .view-price__btn, .footer__btn {
+  @apply text-sm text-white font-bold bg-primary px-10 py-4 rounded-xl hover:bg-[#313852] relative z-10;
+}
+.footer__btn {
+  @apply bg-white text-primary;
+}
+.btn--secondary, .header__btn--secondary {
+  @apply text-primary bg-white border-solid border-light-violet border-[2px] hover:bg-primary hover:text-white;
+}
+.btn--light {
+  @apply text-primary bg-white;
+}
+.btn--full {
+  @apply w-full;
+}
+
+/* list */
+.list, .features__list, .footer__legal-list {
+  @apply grid grid-cols-3 gap-8;
+}
+.testimonials__list {
+  @apply grid grid-cols-3 gap-6;
+}
+.list--gap-sm {
+  @apply gap-6;
+}
+
+/* heading */
+.heading, .discover__item-heading {
+  @apply text-xl font-bold text-primary relative z-10;
+}
+.features__heading, .header__heading, .testimonials__heading, .view-price__heading {
+  @apply  font-bold text-primary relative z-10 text-3xl text-center;
+}
+.header__heading--left {
+  @apply text-left;
+}
+.features__card-heading {
+  @apply font-bold text-primary relative z-10 text-lg;
+}
+.about-us__heading, .features__card-heading--lg {
+  @apply font-bold text-primary relative z-10 text-2xl;
+}
+.cta__heading, .discover__heading, .services__heading {
+  @apply font-bold text-primary relative z-10 text-3xl;
+}
+.testimonials__profile-heading {
+  @apply font-bold text-primary relative z-10 text-lg;
+}
+
+.heading--sm {
+  @apply text-lg;
+}
+.heading--xs {
+  @apply text-sm;
+}
+.heading--md {
+  @apply text-2xl;
+}
+.heading--lg {
+  @apply text-3xl;
+}
+.heading--xl {
+  @apply text-4xl;
+}
+.heading--center {
+  @apply text-center;
+}
+.heading--light {
+  @apply text-white;
+}
+
+/* paragraph  */
+.text,.features__card-text, .about-us__text, .cta__text, .discover__text  {
+  @apply text-sm text-primary relative z-10;
+}
+.header__text, .about-us__item-text, .discover__item-text {
+  @apply text-lg text-primary relative z-10 text-center;
+}
+.testimonials__card-text {
+  @apply text-sm text-primary relative z-10 text-center font-semibold;
+}
+.testimonial__profile-text {
+  @apply text-primary relative z-10 text-xs;
+}
+.view-price__text {
+  @apply text-sm text-primary relative z-10 text-center;
+}
+.footer__legal-text {
+  @apply text-sm  relative z-10 text-[#A5A6AC];
+}
+.services__text {
+ @apply text-sm text-primary relative z-10 opacity-[87%];
+}
+.footer__newsletter-text {
+  @apply text-sm relative z-10 font-semibold text-white;
+}
+.header__text--left {
+  @apply text-left;
+}
+.text--lg {
+  @apply text-lg;
+}
+.text--xl {
+  @apply text-[20px]
+}
+.text--center {
+  @apply text-center;
+}
+.text--bold {
+  @apply font-semibold;
+}
+.text--light {
+  @apply text-white;
+}
+.text--sm {
+  @apply text-xs;
+}
+.text--quote {
+    @apply border-light-violet pl-5 border-solid border-l-2
+}
+.text--light-gray, .features__card-text--light-gray {
+  @apply text-[#A5A6AC];
```

Comment:
define this color and apply it wherever you are using it  same applies to other custom colors